test/integration/org/org_test.go

```diff
@@ -0,0 +1,247 @@
+
+			// compute.requireOsLogin is neither a boolean policy nor a list policy
+			requireOsLogin := gcloud.Runf(t, "resource-manager org-policies describe %s --folder %s", "constraints/compute.requireOsLogin", parentFolder)
+			assert.Equal("constraints/compute.requireOsLogin", requireOsLogin.Get("constraint").String(), "org policy should require OS Login")
+
+			// security command center
+			sccProjectID := org.GetStringOutput("scc_notifications_project_id")
+			topicName := "top-scc-notification"
+			topicFullName := fmt.Sprintf("projects/%s/topics/%s", sccProjectID, topicName)
+			topic := gcloud.Runf(t, "pubsub topics describe %s --project %s", topicName, sccProjectID)
+			assert.Equal(topicFullName, topic.Get("name").String(), fmt.Sprintf("topic %s should have been created", topicName))
+
+			subscriptionName := "sub-scc-notification"
+			subscriptionFullName := fmt.Sprintf("projects/%s/subscriptions/%s", sccProjectID, subscriptionName)
+			subscription := gcloud.Runf(t, "pubsub subscriptions describe %s --project %s", subscriptionName, sccProjectID)
+			assert.Equal(subscriptionFullName, subscription.Get("name").String(), fmt.Sprintf("subscription %s should have been created", subscriptionName))
+
+			orgID := utils.ValFromEnv(t, "TF_VAR_org_id")
+			notificationName := org.GetStringOutput("scc_notification_name")
+			notification := gcloud.Runf(t, "scc notifications describe %s --organization %s", notificationName, orgID)
+			assert.Equal(topicFullName, notification.Get("pubsubTopic").String(), fmt.Sprintf("notification %s should use topic %s", notificationName, topicName))
+
+			//logging
+			billingLogsProjectID := org.GetStringOutput("org_billing_logs_project_id")
+			billingDatasetName := "billing_data"
+			billingDatasetFullName := fmt.Sprintf("%s:%s", billingLogsProjectID, billingDatasetName)
+			billingDataset := gcloud.Runf(t, "alpha bq datasets describe %s --project %s", billingDatasetName, billingLogsProjectID)
+			assert.Equal(billingDatasetFullName, billingDataset.Get("id").String(), fmt.Sprintf("dataset %s should exist", billingDatasetFullName))
+
+			auditLogsProjectID := org.GetStringOutput("org_audit_logs_project_id")
+
+			auditLogsDatasetName := "audit_logs"
+			auditLogsDatasetFullName := fmt.Sprintf("%s:%s", auditLogsProjectID, auditLogsDatasetName)
+			auditLogsDataset := gcloud.Runf(t, "alpha bq datasets describe %s --project %s", auditLogsDatasetName, auditLogsProjectID)
+			assert.Equal(auditLogsDatasetFullName, auditLogsDataset.Get("id").String(), fmt.Sprintf("dataset %s should exist", auditLogsDatasetFullName))
+
+			logsExportStorageBucketName := org.GetStringOutput("logs_export_storage_bucket_name")
+			gcAlphaOpts := gcloud.WithCommonArgs([]string{"--project", auditLogsProjectID, "--json"})
+			bkt := gcloud.Run(t, fmt.Sprintf("alpha storage ls --buckets gs://%s", logsExportStorageBucketName), gcAlphaOpts).Array()[0]
```

Comment:
minor nit: When we just want an additional project arg using `gcloud.Runf` is more readable.

Reply:
fixed in other occurrences in the code.

this one needs the `gcloud.WithCommonArgs` so that we can replace the default `--format json`  with `--json` because of the syntax of the alpha command